Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -1013,7 +1013,20 @@ Right: the mesh with added 1D-features along sharp edges and edges shared
 by at least two of the input polyhedral surfaces forming the complex.
 \cgalFigureCaptionEnd
 
+\subsubsection Mesh_3EdgesSizingField Sizing field for feature edges
 
+The following example shows how to generate a mesh from a polyhedral complex
+or a polyhedral surface with polyline features.
+Polyline features are covered by a set of so-called "protecting balls" which
```

Comment:
Here we speak for the first time about "Protecting Balls" in the User Manual.

Reply:
It's actually described in [this section](https://doc.cgal.org/latest/Mesh_3/index.html#Mesh_3Protectionof0and1dimensionalExposed)